docs/_build/html/_sources/home.rst.txt

```diff
@@ -26,7 +26,7 @@ To make any changes in the code follow the below steps.
 
 2. **About**: About section describes about the Girlscript Foundation and their goals.
 
-.. image:: ./images/homepage/aboutus.JPG
+.. image:: ./images/homepage/aboutus.png
```

Comment:
Always specify the default size of image used

Reply:
okay sir